CHANGELOG.md

```diff
@@ -10,17 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-
-- Add documentation structure and how it works section. ([#412](https://github.com/jina-ai/finetuner/pull/412))
-
-- Add documentation step by step from install to create training data. ([#416](https://github.com/jina-ai/finetuner/pull/416))
+- Add default values for finetuner `HOST` and `JINA_HUBBLE_REGISTRY`. ([#410](https://github.com/jina-ai/finetuner/pull/410))
 
 - Expose arguments `cpu` and `num_workers` in `finetuner.fit`. ([#411](https://github.com/jina-ai/finetuner/pull/411))
 
-- Add default values for finetuner `HOST` and `JINA_HUBBLE_REGISTRY`. ([#410](https://github.com/jina-ai/finetuner/pull/410))
+- Add documentation structure and how it works section. ([#412](https://github.com/jina-ai/finetuner/pull/412))
 
 - Support passing callbacks to the run configuration. ([#415](https://github.com/jina-ai/finetuner/pull/415))
 
+- Add documentation step by step from install to create training data. ([#416](https://github.com/jina-ai/finetuner/pull/416))
+
+- Expose run methods. ([#425](https://github.com/jina-ai/finetuner/pull/425))
```

Comment:
i believe this is a refactor, should put to `changed` section, rest looks good to me!